elixir/apps/web/lib/web/live/sites/new_token.ex

```diff
@@ -255,6 +310,27 @@ defmodule Web.Sites.NewToken do
     value
   end
 
+  defp debian_command_apt_repository do
+    """
+    mkdir --parents /etc/apt/keyrings
+    wget https://artifacts.firezone.dev/apt/key.gpg -O /etc/apt/keyrings/firezone.gpg
```

Comment:
(nit): curl is more widely available than wget - perhaps use curl here?

Reply:
Is it? I googled around a bit and found this: https://qa.debian.org/popcon-graph.php?packages=wget+curl&show_installed=on&want_legend=on&from_date=&to_date=&hlght_date=&date_fmt=%25Y-%25m&beenhere=1

`wget` seems to be the more popular package.